src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -329,6 +329,10 @@ private Frustum createShadowFrustum() {
 
 	@Override
 	public void renderShadows(WorldRendererAccessor worldRenderer, Camera playerCamera) {
+
+		if (IrisVideoSettings.shadowDistance == 0 && renderDistanceMultiplier < 0) {
+			return;
+		}
```

Comment:
This isn't quite right, the shadow map won't be properly cleared by this point AFAIK. We can look more closely into this once I get a chance to look at your ShadowRenderer refactor PR after 1.1.2 is done.

Reply:
Disabled for now.